src/Storage/Commands.Storage/Queue/Cmdlet/RemoveAzureStorageQueueStoredAccessPolicy.cs

```diff
@@ -98,8 +98,6 @@ internal virtual bool ConfirmRemove(string message)
         [PermissionSet(SecurityAction.Demand, Name = "FullTrust")]
         public override void ExecuteCmdlet()
         {
-            WriteWarning("Force parameter in this cmdlet will be removed in a future release.");
```

Comment:
This Force parameter needs to be removed.  Will ping the storage team about this.